backends/redis.go

```diff
@@ -55,99 +55,84 @@ func (e KeyError) Error() string {
 	return "Key not found"
 }
 
-func (r *RedisBackend) newRedisClusterPool() redis.UniversalClient {
+func (r *RedisBackend) Connect() error {
 	redisLogger.Info("Creating new Redis connection pool")
 
-	timeout := 5 * time.Second
-
-	if r.config.Timeout > 0 {
-		timeout = time.Duration(r.config.Timeout) * time.Second
-	}
-
-	var tlsConfig *tls.Config
-	if r.config.UseSSL {
-		tlsConfig = &tls.Config{
-			InsecureSkipVerify: r.config.SSLInsecureSkipVerify,
-		}
-	}
-
-	var client redis.UniversalClient
-	opts := &redis.UniversalOptions{
-		MasterName:       r.config.MasterName,
-		SentinelPassword: r.config.SentinelPassword,
-		Addrs:            r.getRedisAddrs(),
-		DB:               r.config.Database,
-		Username:         r.config.Username,
-		Password:         r.config.Password,
-		IdleTimeout:      240 * time.Second,
-		ReadTimeout:      timeout,
-		WriteTimeout:     timeout,
-		DialTimeout:      timeout,
-		TLSConfig:        tlsConfig,
+	conf := r.config
+	optsR := model.RedisOptions{
+		Username:         conf.Username,
+		Password:         conf.Password,
+		Host:             conf.Host,
+		Port:             conf.Port,
+		Timeout:          conf.Timeout,
```

Comment:
Is timeout behaving the same as before?

Reply:
yes, is defaulting to 5secs if not set and the number represent number of seconds. In the background is also the same library being use go-redis

Reply:
Do we have a replacement for IdleTimeout?

Reply:
So far none of the components is using the idle timeout, if required then we need to expose that config from the storage library and set a value in each tyk component